### PR TITLE
Move resources ids to Ulid with prefix

### PR DESCRIPTION
**Pull request by nanobash**
## Summary

[Preview](https://preview.redoc.ly/all-rebilly/ulid%2Fbatch-2/tag/Files/operation/PostFile)

- [x] Update File resource ids
- [x] Update Attachment resource ids
- [x] Update KycDocument resource ids
- [x] Update KycRequest resource ids
- [x] Update ApiKey resource ids
- [x] Update Fee resource ids
- [x] Update BillingPortal resource ids

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Comment by nanobash** [comment]
@Rebilly/php please review

File: openapi/components/schemas/ApiKey.yaml [hunk 1]
```diff
@@ -4,9 +4,11 @@ required:
   - description
 properties:
   id:
+    type: string
     readOnly: true
-    allOf:
-      - $ref: ./ResourceId.yaml
+    description: Unique resource ID.
```
Comment (swilczynski) [inline openapi/components/schemas/ApiKey.yaml:9]:
```suggestion
    description: ID of the API key.
```

File: openapi/components/schemas/KycRequest.yaml [hunk 2]
```diff
@@ -5,9 +5,11 @@ required:
   - documents
 properties:
   id:
+    type: string
     readOnly: true
-    allOf:
-      - $ref: ./ResourceId.yaml
+    description: Unique resource ID.
```
Comment (swilczynski) [inline openapi/components/schemas/KycRequest.yaml:10]:
```suggestion
    description: ID of the KYC request.
```

File: openapi/components/schemas/Tracking/ApiTracking.yaml [hunk 3]
```diff
@@ -53,8 +53,10 @@ properties:
         maxLength: 50
         example: usr_0YVCEENYJ3D7Q9EN6BN16HA0G4
       apiKeyId:
-        allOf:
-          - $ref: ../ResourceId.yaml
+        type: string
+        description: Unique resource ID.
```
Comment (swilczynski) [inline openapi/components/schemas/Tracking/ApiTracking.yaml:57]:
```suggestion
        description: ID of the API tracking log.
```

File: openapi/components/schemas/Rules/RulesEmailNotification.yaml [hunk 4]
```diff
@@ -35,7 +35,7 @@ properties:
       If your email message templates are localized into more than one language,
       set a customer locale.
 
-      Invalid placeholders render as an empty strings. 
+      Invalid placeholders render as an empty strings.
```
Comment (noon-dawg) [inline openapi/components/schemas/Rules/RulesEmailNotification.yaml:38]: nitpick: `an` is not needed here
```suggestion
      Invalid placeholders render as empty strings.
```

File: openapi/components/schemas/Storefront/StorefrontBillingPortal.yaml [hunk 5]
```diff
@@ -4,10 +4,11 @@ required:
...
   id:
+    type: string
     description: ID of the billing portal.
     readOnly: true
-    allOf:
-      - $ref: ../ResourceId.yaml
+    maxLength: 50
+    example: bill_prt_0YV7K5TYV5D1P9ZNKDT39KZC3C
   token:
     description: Session's token used for authentication.
```
Comment (noon-dawg) [inline openapi/components/schemas/Storefront/StorefrontBillingPortal.yaml:13]: nitpick: [Possessive apostrophes](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#possessive-apostrophes)
```suggestion
    description: Token used for session authentication.
```

File: openapi/components/schemas/Common/CommonKycDocument.yaml [hunk 6]
```diff
@@ -6,14 +6,17 @@ required:
   - status
 properties:
   id:
+    type: string
     readOnly: true
-    allOf:
-      - $ref: ../ResourceId.yaml
+    description: Unique resource ID.
```
Comment (noon-dawg) [inline openapi/components/schemas/Common/CommonKycDocument.yaml:11]: nitpick: we should use this format (I'm not if my attempt has the correct meaning here)
```suggestion
    description: ID of the KYC document.
```

File: openapi/components/schemas/Files/File.yaml [hunk 7]
```diff
@@ -1,9 +1,11 @@
 type: object
 properties:
   id:
+    type: string
+    description: File ID.
```
Comment (noon-dawg) [inline openapi/components/schemas/Files/File.yaml:5]: nitpick: we should use this format
```suggestion
    description: ID of the file.
```

**Comment by nanobash** [comment]
@Rebilly/php please review

File: openapi/components/schemas/Attachment.yaml [hunk 8]
```diff
@@ -5,12 +5,16 @@ required:
   - relatedType
 properties:
   id:
+    type: string
     readOnly: true
-    allOf:
-      - $ref: ./ResourceId.yaml
+    description: Attachment ID.
```
Comment (lexerom) [inline openapi/components/schemas/Attachment.yaml:10]:
```suggestion
    description: ID of the attachment.
```

**Review by noon-dawg — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**